src/components/pages/search-pages/subject-search/components/SubjectSearchList.tsx

```diff
@@ -24,10 +24,10 @@ export const SubjectSearchList = ({ subjects }: GetListOfSubjectsResponse) => {
   };
 
   return (
-    <ul className={styles['subject-search-list']}>
+    <Box component="ul" sx={styles.subjectList}>
```

Comment:
mui has elements for list, you can check them

Reply:
I can't find(